src/lib/viz/popups/Popup.ts

```diff
@@ -303,16 +294,8 @@ export interface PopupElement {
    * Title for this element.
    */
   title?: string | null;
-
-  /**
-   * d3 format for the value of this attribute.
-   */
-  format?: string | FormatFunction;
```

Comment:
we should keep the `format` option for functions right?

```javascript
format?: (value: any) => any;
```

Reply:
I thought it was just related to d3... oka

Reply:
Yup sorry, the comment was outdated and wrong :sweat_smile: